docs/changelog/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * FEATURE: all VictoriaMetrics [enterprise](https://docs.victoriametrics.com/enterprise/) components: add support of hot-reload for license key supplied by `-licenseFile` command-line flag.
 * FEATURE: [vmgateway](https://docs.victoriametrics.com/vmgateway/): allow disabling `Bearer` prefix enforcement for authentication header. This is useful for cases when identity token is used instead of access token.
 * FEATURE: [vmgateway](https://docs.victoriametrics.com/vmgateway/): support parsing `vm_access` claims in string format. This is useful for cases when identity provider does not support mapping claims to JSON format.
+* FEATURE: `vmstorage` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): automatically calculate the `maxUniqueTimeseries` based on available memory and concurrent requests if it is not set. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6930).
```

Comment:
I think this change should be also setting `maxUniqueTimeseries ` default value to 0 on vmselect.
The idea is that vmselect shouldn't tell vmstorage what the limit is, by default.
And if user set this on vmselect, this should be used as a soft limit - never override the vmstorage default or custom limit.

Reply:
This change should be also reflected in https://docs.victoriametrics.com/#resource-usage-limits and `maxUniqueTimeseries ` flag description